### DISCUSSION TITLE
Is there a way to quiet or suppress the (unchanged) output lines?

### DISCUSSION THREAD

**Discussion by jebarpg** [prettier/prettier#15981] [Q&A] [answered]
Pretty much just don't want to see a message if a file was not changed by prettier. i.e. quiet messages like this:
```
package-lock.json 360ms (unchanged)
package.json 48ms (unchanged)
public/404.html 129ms (unchanged)
public/css/style.css 51ms (unchanged)
```

**Comment by your-diary** [comment, marked as answer]
Any updates?

My dirty workaround: `npm run format | rg -v -F '(unchanged)'`

**Comment by ZaLiTHkA** [comment]
I fully agree with this... my opinion of this "opinionated code formatter" is that the output of `--write` should be more like that of `--check`, where it should only print a message for files that were updated, not all files checks with a suffix to state whether or not it was updated.

I've lost countless hours of my life trying to switch to a different code formatter, but I always keep coming back to Prettier. then I see it's inconsistent feedback between `--check` and `--write`, and suddenly I'm right back where I started, looking for something else.

having said that, changing the default behaviour could be considered a "potential breaking change" though, as there could be systems out there that depend on the output of this command for automation purposes... 🤔

could one of the project maintainers perhaps chime in on this topic?

**Comment by ekaj2** [comment]
Use `--write --list-different`

**Reply by miguno** [reply]
Thanks!

**Reply by Starefossen** [reply]
This ought to be the default imho
